<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWidget</class>
 <widget class="QFrame" name="ProgressWidget">
  <property name="frameStyle">
   <set>QFrame::NoFrame</set>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_0">
   <item>
    <widget class="QPushButton" name="openPushButton">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
